test/serializer/basic/FunctionUndelayOrdering.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+var f = function(obj) {
+  return function() {
+    /*This comment will make the function too big to inline*/
+    return obj;
+  }
+}
+var obj1 = {};
+var obj2 = {};
+var g1 = f(obj1);
+obj1.foo = g1;
+var g2 = f(obj2);
+obj1.bar = g2;
+inspect = function() {  return obj1.foo(); };
```

Comment:
This will return an object, and we'll probably compare the toString of that object. That seems a bit brittle, as we'll compare that output from node.js against the output of Prepack.